django/db/migrations/graph.py

```diff
@@ -126,33 +126,66 @@ def clear_cache(self):
                 self.node_map[node].__dict__.pop('_descendants', None)
             self.cached = False
 
-    def forwards_plan(self, node):
+    def forwards_plan(self, target):
```

Comment:
Any chance to separate the rename of `node` to `target` from the commit adding the iterative fallback. I suggest to first rename `node` to `target` and then add the iterative fallback.